lib/slack-ruby-bot-server/api/endpoints/teams_endpoint.rb

```diff
@@ -60,6 +61,8 @@ class TeamsEndpoint < Grape::API
               )
             end
 
+            team.state = params[:state] if params[:state]
+
             Service.instance.create!(team)
```

Comment:
I think you want to change `create!` to take `options = {}` in which you pass in whatever was in `params` for future compatibility.